dbms/src/Interpreters/Join.h

```diff
@@ -22,6 +22,7 @@
 #include <Core/Spiller.h>
 #include <DataStreams/IBlockInputStream.h>
 #include <DataStreams/RuntimeFilter.h>
+#include <Flash/Coprocessor/DAGContext.h>
```

Comment:
useless?

Reply:
No, `JoinProfileInfo` struct is moved to `DAGContext.h`